script.module.pyserial/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
License file should be the same the module provides: https://github.com/pyserial/pyserial/blob/master/LICENSE.txt